addons/web/static/src/legacy/js/widgets/data_export.js

```diff
@@ -265,7 +265,7 @@ var DataExport = Dialog.extend({
         this.$records = this.$('.o_export_tree_item');
         this.$records.each(function (i, el) {
             var $el = $(el);
-            $el.find('.o_tree_column').first().toggleClass('o_required', !!self.records[$el.data('id')].required);
+            $el.find('.o_tree_column').first().toggleClass('o_required border-bottom border-dark', !!self.records[$el.data('id')].required);
```

Comment:
It doesn't achieve the desired result, the border seems a glitch :/

<img width="489" alt="image" src="https://user-images.githubusercontent.com/8558351/162149701-cd4766ed-233e-47fe-a975-6ccc33c716d9.png">

What about setting it to `font-weight-bolder` instead? 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/8558351/162150053-b935e855-f0de-49f1-bd03-d5898d5ab39c.png">

Reply:
I definitely like more the font-weight-bolder result. Will update.